### PR TITLE
[Toolchain] Use installed toolchain functions

### PR DESCRIPTION
**Pull request by jyoungyun**
This commit uses installed toolchain getter function.

ONE-vscode-DCO-1.0-Signed-off-by: Jiyoung Yun <jy910.yun@samsung.com>

### CODE REVIEW

**Review by hyunsik-yoon — DISMISSED** [review]

**Comment by yunjayh** [comment]
This fails `Toolchian.listAvailable` test. I think you need to add a change to the test code, too (maybe?).

File: src/Toolchain/ToolchainEnv.ts [hunk 1]
```diff
@@ -125,11 +125,22 @@ class ToolchainEnv extends Env {
...
+    return this.compiler.getToolchainTypes()
+        .map((type) => this.compiler.getToolchains(type, 0, 20))
+        .reduce((r, a) => {
+          return r.concat(a);
+        });
   }
 
-  listInstalled(): Toolchain|undefined {
-    return this.installed;
+  listInstalled(): Toolchain {
```
Comment (YongseopKim) [inline src/Toolchain/ToolchainEnv.ts:136]: Toolchain or undefined? or just Toolchain?

**Comment by hyunsik-yoon** [comment]
FYI,
@jyoungyun Later, we may need to show toolchain information (e.g., dependent package list) when user clocks or hovers mouse over a toolchain node in the view.

One approach we could consider might be markdown tooltip.

```typescript
tooltip?: string | MarkdownString | undefined;
```

example: 
- https://github.com/microsoft/vscode/issues/100741#issuecomment-668642462

<src img="https://user-images.githubusercontent.com/38270282/89308368-78d40c80-d672-11ea-8248-6ed0d0cb0b2f.gif" />

/cc @dayo09 @struss

**Review by hyunsik-yoon — APPROVED** [review]

**Comment by jyoungyun** [comment]
/cc @hyunsik-yoon 

>One approach we could consider might be markdown tooltip.

Oh, I already used tooltip for dependency information.
Here is draft PR(#688).

**Review by YongseopKim — APPROVED** [review]
LGTM

**Review by yunjayh — APPROVED** [review]
LGTM

**Pull request merged**